setup.py

```diff
@@ -12,12 +12,13 @@
 
 install_requirements = [
     "boto3>=1.3,<2.0",
-    "click==6.7",
+    "click>=7.0<8.0",
```

Comment:
I am not sure what the consequences of upgrading this package by 2 major version releases will be. Prefer to keep `>=6.7,<7.0`. Until we can do further test

Reply:
then it will still conflict with awscli and other libs